src/App.js

```diff
@@ -1,8 +1,8 @@
-import React from 'react';
-import AppHeader from './components/AppHeader/AppHeader';
-import AppContent from './components/AppContent/AppContent';
+import React from "react";
+import AppHeader from "./components/AppHeader/AppHeader";
+import AppContent from "./components/AppContent/AppContent";
```

Comment:
There is a better approach for components naming in the [style guide](https://github.com/airbnb/javascript/tree/master/react).
![image](https://user-images.githubusercontent.com/52723192/190529993-4e2647ac-a404-41a5-b38e-6c51fbf6cfbc.png)